app/models/like.rb

```diff
@@ -1,4 +1,9 @@
 class Like < ApplicationRecord
   belongs_to :user
```

Comment:
- [ ] @sambeck87 kindly add a class name as you did in the comment and post models